.env.template

```
@@ -94,20 +94,20 @@ MODELS=`[
       ]
     },
     {
-      "name": "mistralai/Mistral-7B-Instruct-v0.1",
-      "displayName": "mistralai/Mistral-7B-Instruct-v0.1",
-      "description": "Mistral 7B is a new Apache 2.0 model, released by Mistral AI that outperforms Llama2 13B in benchmarks.",
-      "websiteUrl": "https://mistral.ai/news/announcing-mistral-7b/",
+      "name": "HuggingFaceH4/zephyr-7b-alpha",
+      "displayName": "HuggingFaceH4/zephyr-7b-alpha",
+      "description": "Zephyr 7B α is a fine-tune of Mistral 7B, released by the Hugging Face H4 RLHF team.",
+      "websiteUrl": "https://huggingface.co/HuggingFaceH4/zephyr-7b-alpha/",
       "preprompt": "",
-      "chatPromptTemplate" : "<s>{{#each messages}}{{#ifUser}}[INST] {{#if @first}}{{#if @root.preprompt}}{{@root.preprompt}}\n{{/if}}{{/if}}{{content}} [/INST]{{/ifUser}}{{#ifAssistant}}{{content}}</s>{{/ifAssistant}}{{/each}}",
+      "chatPromptTemplate" : "<|system|>\n{{preprompt}}</s>\n{{#each messages}}{{#ifUser}}<|user|>\n{{content}}</s>\n<|assistant|>\n{{/ifUser}}{{#ifAssistant}}{{content}}</s>\n{{/ifAssistant}}{{/each}}",
       "parameters": {
-        "temperature": 0.1,
+        "temperature": 0.7,
         "top_p": 0.95,
         "repetition_penalty": 1.2,
         "top_k": 50,
         "truncate": 1000,
         "max_new_tokens": 2048,
```

Maybe 32, or even 16? 

```suggestion
        "max_new_tokens": 32,
```

---

We offer it as a chat model as well, so I wouldn't put a limit here

---

oh yes my bad it shoud probably be another conf